### PR TITLE
Updating wrapper for MOI 0.6

### PR DESCRIPTION
**Pull request by vitornesello**
Tests linear15 were disables because they are using `VectorAffineFunction`.
All tests are passing with MOI 0.6 in OSX using Julia 0.6 and Julia 0.7

### CODE REVIEW

**Comment by vitornesello** [comment]
This is the follow up to #67

File: src/MOIWrapper.jl [hunk 1]
```diff
@@ -136,30 +170,38 @@ function update_bounds_for_binary_vars!(col_lb::Vector{Float64}, col_ub::Vector{
...
 end
 
 """
-    function copy!(cbc_optimizer, user_optimizer; copynames=false)
+    function copy_to(cbc_optimizer, user_optimizer; copy_names=false)
 
-Receive a cbc_optimizer which contains the pointer to the cbc C object and instantiate the object cbc_model_format::CbcModelFormat based on user_optimizer::AbstractModel (also provided by the user).
-Function loadProblem of CbcCInterface requires all information stored in cbc_model_format.
+Receive a cbc_optimizer which contains the pointer to the cbc C object and 
+instantiate the object cbc_model_format::CbcModelFormat based on
```
Comment (mlubin) [inline src/MOIWrapper.jl]: This documentation is a bit confusing. Aren't these implementation details that callers of `copy_to` don't need to know?

Comment (vitornesello) [reply src/MOIWrapper.jl]: you are right, i removed it

File: src/MOIWrapper.jl [hunk 2]
```diff
@@ -136,30 +170,59 @@ function update_bounds_for_binary_vars!(col_lb::Vector{Float64}, col_ub::Vector{
...
     end
 end
 
-function update_integer_indices(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
-    ci::Vector{MOI.ConstraintIndex{F,S}}, integer_indices::Vector{Int}) where {F, S}
-    for i in 1:length(ci)
-        f = MOI.get(user_optimizer, MOI.ConstraintFunction(), ci[i])
-        push!(integer_indices, mapping.varmap[f.variable].value)
+function create_indices_for_bounds(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
+                                   zeroone_indices::Vector{Int}, integer_indices::Vector{Int})
```
Comment (mlubin) [inline src/MOIWrapper.jl]: Naming nit: `zero_one_indieces`

File: src/MOIWrapper.jl [hunk 3]
```diff
@@ -136,30 +170,59 @@ function update_bounds_for_binary_vars!(col_lb::Vector{Float64}, col_ub::Vector{
...
+        push!(list_of_indices, mapping.varmap[f.variable].value)
     end
 end
 
-function update_integer_indices(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
-    ci::Vector{MOI.ConstraintIndex{F,S}}, integer_indices::Vector{Int}) where {F, S}
-    for i in 1:length(ci)
-        f = MOI.get(user_optimizer, MOI.ConstraintFunction(), ci[i])
-        push!(integer_indices, mapping.varmap[f.variable].value)
+function create_indices_for_bounds(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
```
Comment (mlubin) [inline src/MOIWrapper.jl]: I don't immediately understand what this function does. A full docstring isn't needed, but a comment would help.

File: src/MOIWrapper.jl [hunk 4]
```diff
@@ -136,30 +170,59 @@ function update_bounds_for_binary_vars!(col_lb::Vector{Float64}, col_ub::Vector{
...
+                mapping.conmap[list_of_ci[i]] = MOI.ConstraintIndex{F,S}(num_bounds + i)
+            end
+            num_bounds += MOI.get(user_optimizer, MOI.NumberOfConstraints{F,S}())
+        end
     end
 end
 
-"""
-    function copy!(cbc_optimizer, user_optimizer; copynames=false)
+function create_indices_for_rows(cbc_optimizer::CbcOptimizer, user_optimizer::MOI.ModelLike,
```
Comment (mlubin) [inline src/MOIWrapper.jl]: Same thing, please add a comment or docstring explaining what this function does.

File: src/MOIWrapper.jl [hunk 5]
```diff
@@ -171,51 +234,31 @@ function MOI.copy!(cbc_optimizer::CbcOptimizer,
...
-            ## need to add a row
-            for i in 1:length(ci)
-                mapping.conmap[ci[i]] = MOI.ConstraintIndex{F,S}(num_rows + i)
-            end
-            num_rows += MOI.get(user_optimizer, MOI.NumberOfConstraints{F,S}())
-        end
-    end
+    create_indices_for_rows(cbc_optimizer, user_optimizer, mapping) # Updates mapping
+    num_rows = length(mapping.conmap)
+    # Do not create rows for bounds, but has to update mapping
```
Comment (mlubin) [inline src/MOIWrapper.jl]: Nit (full sentences): "Rows are not created for bounds, but the mapping is still updated."

**Comment by mlubin** [comment]
Thanks!

**Pull request merged**